src/src/components/Login.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <form class="login" @submit.prevent="handleLogin">
+    <img src="../assets/images/vaultN-logo.svg" class="login-logo" />
+    <h1>{{ contentLogin.loginHeadlineText }}</h1>
+    <CustomInput :input="customEmailInput" @valueChanged="handleTextChanged"/>
+    <CustomInput :input="customPasswordInput" @valueChanged="handleTextChanged"/>
+    <FormErrors v-if="errroMessages" :messages="errroMessages"></FormErrors>
+    <a href="#/forgot" class="link">{{ contentLogin.loginForgotPasswordText }}</a>
+    <CustomButton :button="customLoginButton" @buttonClicked="handleLogin"/>
+    <!-- <button class="btn btn-login" type="submit">{{ contentLogin.loginButtonText }}</button> -->
+    <a href="#/reset" class="link">{{ contentLogin.loginCreateAccountText }}</a>
+  </form>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  ref,
+} from 'vue';
+import store from '../store/index';
+import FormErrors from './form/FormErrors.vue';
+import CustomInput from './form/CustomInput.vue';
+import CustomButton from './form/CustomButton.vue';
+import InputModel from '../models/InputModel';
+import ButtonModel from '../models/Buttonmodel';
+import MessageModel from '../models/MessageModel';
+
+export default defineComponent({
+  name: 'Login',
+  components: {
+    FormErrors,
+    CustomInput,
+    CustomButton,
+  },
+  setup() {
+    const email = ref('');
+    const password = ref('');
+    const errroMessages = ref<MessageModel[]>([]);
+    const contentLogin = computed(() => store.getters.contentLogin);
+    const customEmailInput = ref<InputModel>({
+      customClass: 'input-email',
+      fieldIcon: 'email',
+      fieldPlaceholder: 'Email',
+      disabled: false,
+      suffix: '*',
+      prefix: '',
+      fieldName: 'email',
+      fieldValue: '',
+      readOnly: false,
+    });
+    const customPasswordInput = ref<InputModel>({
+      customClass: 'input-password',
+      fieldIcon: 'password',
+      fieldPlaceholder: 'Password',
+      disabled: false,
+      suffix: '*',
+      prefix: '',
+      fieldName: 'password',
+      fieldValue: '',
+      readOnly: false,
+    });
+    const customLoginButton = ref<ButtonModel>({
+      customClass: 'btn-login',
+      buttonIcon: '',
+      disabled: false,
+      suffix: '*',
+      prefix: '',
+      buttondName: 'login',
+      value: contentLogin.value.loginButtonText,
+      readOnly: false,
+    });
+    function handleLogin() {
+      console.log('TODO: Fix login');
+    }
+    function handleTextChanged() {
+      console.log('TODO: Fix Text changed');
+    }
+    return {
+      email,
+      password,
+      contentLogin,
+      errroMessages,
+      customEmailInput,
+      customPasswordInput,
+      customLoginButton,
+      handleLogin,
+      handleTextChanged,
+    };
+  },
+});
+</script>
+
+<style scoped lang="scss">
```

Comment:
Maybe use separate scss files? This will avoid large Vue files and keep a more structured and separated approach.